templates/taxi/driver_license_update_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}

```

Comment:
you don't need to create an extra template for this. You can just use yes|no on driver_form